articles/integrations/office-365.md

```diff
@@ -8,7 +8,7 @@ Your users will be able to log in using their existing Active Directory credenti
 
 This would typically require you to setup [an advanced ADFS infrastructure](https://msdn.microsoft.com/en-us/library/azure/dn151324.aspx) with Federation Servers in the corporate network and Web Application Proxies exposed in the DMZ. But with Auth0 as an identity provider for Office 365 all of this is handled by the [AD Connector](/connector/overview) which doesn't require you to expose any of your servers to the outside world.
 
-In addition to that users can also have SSO with other applications they're building or Third Party Applications they're using (like Salesforce, Dropbox, SharePoint Server, ...).
+In addition to that users can also have SSO Integrations with other applications they're building or third party applications they're using (like Salesforce, Dropbox, SharePoint Server, ...).
```

Comment:
This isn't right. Change to:

`In addition to that users can also setup SSO with custom apps or integrations like Salesforce, Dropbox, SharePoint Server, ....`